### ISSUE TITLE
Show hours in timestamps if total document length is > 1h

### ISSUE THREAD

**Issue by pajowu** [transcribee/transcribee#163]
Currently we decide whether to show the hour of a timestamp based on the timestamp (if t>1h, we show it). This leads to weird results for documents with >1h length: In the first part of the document, the hours are not shown, in the second part they are. Once #144 is merged, we can easily check the total duration of a document and decide whether to show the hours based on this

_Originally posted by @rroohhh in https://github.com/transcribee/transcribee/pull/161#discussion_r1194723506_

**Comment by pajowu** [comment]
No longer needed since #165
